packages/tracking/src/enableTracking.ts

```diff
@@ -22,7 +22,7 @@ import { config, gtag } from './shared';
 
 const SCRIPT_IDENTIFIER = 'data-web-stories-tracking';
 
-function loadScriptTag(url) {
+function loadScriptTag(url: string) {
```

Comment:
Might want to add the `Promise<void>` return type here

Reply:
Technically it's `Promise<Event>` but yes, good idea 👍